build.gradle

```diff
@@ -1,3 +1,6 @@
+def configDocDir = "${buildDir}/configDocs"
```

Comment:
Doing a clean build becomes more problematic if files generated by the build are not under the build directory.  As it stands now I can clean the build by just removing one directory.  What does having more than one directory buy us?